CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins
 
 ### Fixed
 - minor typos in PR template (@majormoses)
+- handler-graphite-status.rb: use proper rescue class (@MrMisa93)
```

Comment:
3.1.0 has already released it needs to be added under `## [Unreleased]` like so:
```
## [Unreleased]
### Fixed
- handler-graphite-status.rb: use proper rescue class (@MrMisa93)
```

In this case I will fix it up for you and work on cutting a release this evening.